samples/Samples.Jwt/JwtHelper.cs

```diff
@@ -85,8 +85,7 @@ private static (SecurityKey SecurityKey, string SecurityAlgorithm) CreateSymmetr
     {
         // hash the password and use that to create a symmetric key for signing the JWT tokens
         var passwordBytes = System.Text.Encoding.UTF8.GetBytes(password);
-        using var sha = SHA256.Create();
-        var keyBytes = sha.ComputeHash(passwordBytes);
+        var keyBytes = SHA256.HashData(passwordBytes);
```

Comment:
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1850

Reply:
Ok, although note that this is only valid in .NET 5+.  Works here because this is a sample that targets .NET 7.  I'm okay with the change.